euclidean_gcd/euclidean_gcd.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import "fmt"
+
+// EuclideanGCDIterative finds GCD of given numbers by iterative method
+// first : first number
+// second : second number
+// tmp : temporary variable which hold values
+// Time complexity :O(log min(first,second))
+func EuclideanGCDIterative(first int, second int) int {
+	for second != 0 {
+		tmp := first
+		second = first % second
+		first = tmp
+	}
+	return first
+}
+
+// EuclideanGCDRecursive finds GCD of given numbers by recursion
+// first : first number
+// second : second number
+// Recurrance equation : gcd(first,second) =gcd(second,first mod second),if b!=0 gcd(first,second)=first,if b=0
+// Time complexity : O(log min(first,second))
```

Comment:
@yashLadha According to Go documentation, @rashinanwani has used correct convention. So, I think there is no need to use multi-line comments.